sklearn/metrics/tests/test_score_objects.py

```diff
@@ -285,7 +289,9 @@ def test_classification_scores():
     clf.fit(X_train, y_train)
 
     for prefix, metric in [('f1', f1_score), ('precision', precision_score),
-                           ('recall', recall_score)]:
+                           ('recall', recall_score),
+                           ('jaccard', partial(jaccard_score,
```

Comment:
We don't need the partial here?

Reply:
probably no longer needed

Reply:
comment not addressed?

Reply:
Sorry, went looking in the wrong file and couldn't find it so thought it was fixed. Done now.